finite_difference/src/field_mod.f90

```diff
@@ -215,25 +226,30 @@ function r2d_field_constructor(grid,    &
        ntiley = 1
     end if
 
-    nthreads = 1
-    if (present(do_tile)) then
-        if (do_tile) then
-  !$       nthreads = omp_get_max_threads()
-        endif
-    endif
+    if (local_do_tiling) then
+       nthreads = 1
+!$     nthreads = omp_get_max_threads()
```

Comment:
Is this commented intentionally or just for debugging? So if I understand correctly, right now even if local_do_tiling is True it will just create 1 tile (which is equivalent to no tiling at all)?

Reply:
It's a "special" comment: `!$` means that this line should be compiled when OpenMP is enabled but is ignored otherwise.